src/java/com/zimbra/graphql/models/inputs/GQLPrefInput.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra GraphQL Extension
+ * Copyright (C) 2018 Synacor, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software Foundation,
+ * version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ * ***** END LICENSE BLOCK *****
+ */
+package com.zimbra.graphql.models.inputs;
+import com.zimbra.common.gql.GqlConstants;
+
+import io.leangen.graphql.annotations.GraphQLInputField;
+import io.leangen.graphql.annotations.GraphQLNonNull;
+import io.leangen.graphql.annotations.GraphQLQuery;
+import io.leangen.graphql.annotations.types.GraphQLType;
+
+/**
+ * The GQLPrefInput class.<br>
+ * Contains prefs which are to be modified.
+ * @see com.zimbra.soap.account.message.Pref
+ *
+ * @author Zimbra API Team
+ * @package com.zimbra.graphql.models.inputs
+ * @copyright Copyright © 2018
+ */
+@GraphQLType(name=GqlConstants.CLASS_ACCOUNT_PREF_INPUT, description="Input for prefs.")
+public class GQLPrefInput {
+
+    String name;
+    String value;
+    Long modifiedTimestamp;
+
+    @GraphQLQuery(name=GqlConstants.NAME, description="The name of the pref to get")
+    public String getName() {
+        return name;
+    }
+
+    @GraphQLInputField(name=GqlConstants.NAME, description="The name of the pref to set")
+    public void setName(@GraphQLNonNull String name) {
+        this.name = name;
+    }
+
+    @GraphQLQuery(name=GqlConstants.VALUE, description="The value for the pref to get")
```

Comment:
This description doesn't make sense. Try `The preference value`.

Reply:
Actually reading over the resolver, you never use this class in a response? So you should remove the query annotations in this class since it's never a response.